modules/xphoto/CMakeLists.txt

```diff
@@ -1,2 +1,2 @@
 set(the_description "Addon to basic photo module")
-ocv_define_module(xphoto opencv_core opencv_imgproc WRAP python)
+ocv_define_module(xphoto opencv_core opencv_imgproc opencv_highgui WRAP python)
```

Comment:
Thank you very much for pointing it out. You are right. `highgui` was used originally for showing the processed images. I will remove this dependency to the sample file.